### DISCUSSION TITLE
Suggest Home page change

### DISCUSSION THREAD

**Discussion by mmuhunn** [mmuhunn/CampusShare#29] [Ideas]
The home page seems to have too much text, which makes it harder to read. How about redesigning it to make it more visually appealing and easier to navigate?🙄

**Comment by hellinii** [comment]
I completely agree with the need for a redesign to improve the homepage's usability. I've shared my proposed design direction in the GitHub issue, including an example layout. Please take a look at the issue and let me know your thoughts or suggestions!
